src/gmt_init.c

```diff
@@ -10016,8 +10017,19 @@ void gmt_set_undefined_defaults (struct GMT_CTRL *GMT, double plot_dim, bool con
 		auto_scale = true;
 		if (conf_update) GMT_keyword_updated[GMTCASE_MAP_ANNOT_OFFSET_SECONDARY] = true;
 	}
-	if (gmt_M_is_dnan (GMT->current.setting.map_label_offset)) {
-		GMT->current.setting.map_label_offset = 6 * pt * scale;	/* 6p */
+	if (gmt_M_is_dnan (GMT->current.setting.map_label_offset[GMT_X])) {
+		if (GMT->current.setting.map_label_mode[GMT_X] == GMT_LABEL_AXIS)
+			GMT->current.setting.map_label_offset[GMT_X] = 32 * pt;
```

Comment:
```suggestion
			GMT->current.setting.map_label_offset[GMT_X] = 32 * pt * scale;
```
I set this as constant intentionally, but in retrospect scaled might be better.

Reply:
Do you still want this suggestion to be active?